src/AnalyzerConfiguration/NI.CSharp.Analyzers.TestUtilities.globalconfig

```diff
@@ -0,0 +1,17 @@
+# NOTE: Requires **VS2019 16.3** or later
```

Comment:
According to [this issue](https://github.com/dotnet/roslyn/issues/47707), VS2019 16.7 is required for global config